packages/tizen_rpc_port/example/client/lib/message_client.dart

```diff
@@ -65,7 +65,7 @@ class _NotifyCallback extends _Delegate {
     final String sender = parcel.readString();
     final String message = parcel.readString();
 
-    callback?.call(sender, message);
+    (callback as NotifyCallback)(sender, message);
```

Comment:
@upple Modifications made to auto-generated `message_client.dart` and `message_server.dart`. Please review all and update the TIDL compiler accordingly.

Reply:
https://review.tizen.org/gerrit/c/platform/core/appfw/tidl/+/289709
I'm preparing the patch.

Reply:
Check **ALL** the changes but not just this line.

https://github.com/flutter-tizen/plugins/pull/543/files